_glaredb/sql-commands/copy-to.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: COPY TO
+parent: SQL commands
+---
+
+# COPY TO
+
+Copy a query result to a file.
+
+GlareDB currently only supports creating local copies using this command.
```

Comment:
We do support GCS and S3 as well, just that the URL support is lacking at this state. This will be fixed in the next release.

Reply:
Thanks, I'll give it a QA and only document it if it's stable. There's no harm in documenting it next release. I appreciate this heads up :+1: